### PR TITLE
feat: implement find useful block

### LINKED ISSUE #59: implement find useful block
Image

For instance:
- https://www.delta.com/us/en/skymiles/overview
- https://www.delta.com/us/en/skymiles/how-to-use-miles/travel-with-miles
- https://www.delta.com/us/en/skymiles/airline-credit-cards/plan-it-by-american-express
- https://www.delta.com/us/en/skymiles/airline-credit-cards/takeoff-15

### PR DESCRIPTION
**Pull request by askayastha22**
Fix #59 

Test URLs:
- Before: https://main--delta--hlxsites.hlx.page/us/en/skymiles/overview
- After: https://issue-59--delta--hlxsites.hlx.page/us/en/skymiles/overview

### CODE REVIEW

**Review by ramboz — COMMENTED** [review]
I asked the delta team to provide backend details so we can actually send the feedback to their system

File: us/en/skymiles/blocks/feedback/feedback.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  spanAckQuestion.textContent = 'Did you find the information you were looking for on this page?';
+
+  const divFeedbackButtonsContainer = document.createElement('div');
+  divFeedbackButtonsContainer.classList.add('feedback-buttons-container');
+
+  // Yes button
+  const divYesButton = document.createElement('div');
+  divYesButton.classList.add('feedback-button');
+
+  const yesButton = document.createElement('a');
```
[the remark this answers is not in the document]
Comment (askayastha22) [reply us/en/skymiles/blocks/feedback/feedback.js:38]: > I asked the delta team to provide backend details so we can actually send the feedback to their system

I have investigated what happens when you click the "Yes" and "No". They send the click events to their report suite in Adobe Analytics.

[the remark this answers is not in the document]
Comment (ramboz) [reply us/en/skymiles/blocks/feedback/feedback.js:38]: Oh, cool. That makes it easy then 👍🏼

File: us/en/skymiles/blocks/feedback/feedback.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  inputSurveyID.setAttribute('id', 'surveyID');
+  inputSurveyID.setAttribute('name', 'surveyID');
+  inputSurveyID.setAttribute('value', 'Y0LLcnRhFy9KjxsjyPybemlqYF1spmda');
+
+  const divQuestionContainer = document.createElement('div');
+  divQuestionContainer.classList.add('question-container');
+
+  const spanAckQuestion = document.createElement('span');
+  spanAckQuestion.classList.add('acknowledge-question');
+  spanAckQuestion.textContent = 'Did you find the information you were looking for on this page?';
```
Comment (ramboz) [inline us/en/skymiles/blocks/feedback/feedback.js:29]: Is this always the same message? I wonder if we should let the author potentially configure the question in the block content.

**Review by ramboz — COMMENTED** [review]
I would suggest renaming this all to "find-useful" or similar, since there will be a separate `feedback` widget we'll add later. Just to avoid confusing the 2

File: us/en/skymiles/blocks/feedback/feedback.js (hunk 2 above)
Comment (askayastha22) [reply us/en/skymiles/blocks/feedback/feedback.js:29]: @ramboz I thought about this. Yeah, may be we could take the approach of the `TOC` block. There would be a default message, which they could override. What do you think?

Comment (ramboz) [reply us/en/skymiles/blocks/feedback/feedback.js:29]: Sounds like a great idea!

File: us/en/skymiles/blocks/find-useful/find-useful.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
+function showAcknowledgeText() {
+  // Replace question with acknowledge text
+  const acknowledgeText = document.getElementById('acknowledge-text');
+  const questionContainer = document.querySelector('.question-container');
+  questionContainer.setAttribute('hidden', 'true');
```
Comment (ramboz) [inline us/en/skymiles/blocks/find-useful/find-useful.js:5]: You could use `aria-hidden` attribute instead so we have an accessible property here

**Review by ramboz — APPROVED** [review]
Looks good overall, left a few minor comments

**Comment by askayastha22** [comment]
@ramboz I have incorporated your suggestions.

**Review by ramboz — APPROVED** [review]

**Pull request merged**